Sources/MapboxNavigation/DestinationOptions.swift

```diff
@@ -1,25 +1,43 @@
 import CoreLocation
 import MapboxDirections
 
-// :nodoc:
+/**
+ Customization options for the destination(s) preview using `DestinationPreviewViewController` banner.
+ */
 public struct DestinationOptions {
     
-    // :nodoc:
+    /**
+     Primary text that is going to be shown in `DestinationPreviewViewController`. In case if
+     last waypoint in `DestinationOptions.waypoints` contains non-nil value `Waypoint.name` it'll be
+     used.
+     */
     public var primaryText: NSAttributedString?
     
-    // :nodoc:
+    /**
+     Array of waypoints which are presented.
+     */
     public let waypoints: [Waypoint]
```

Comment:
Should these properties be `var`? This would make it possible for the developer to configure a `DestinationOptions` instance in multiple statements.

Reply:
Would rather keep them as unmodifiable properties, for now there will be only a way to modify `primaryText` in `DestinationOptions`.